### DISCUSSION TITLE
[Question] How to change the background color of an AlertDialog

### DISCUSSION THREAD

**Discussion by vindevoy** [flet-dev/flet#2519] [Q&A]
### Question

At this moment, my alert dialogs have blue-ish color.  I have tried to change that color, but I can't find how.

I have changed the surface_tint_color, overlay_color, bgcolor.  None give an error, but none have effect when I set them.

### Code sample

```python
import flet as ft

def main(page: ft.Page):
    page.title = "AlertDialog question: how can I make this dialog RED for example"

    dlg = ft.AlertDialog(
        title=ft.Text("Hello, you!"), on_dismiss=lambda e: print("Dialog dismissed!")
    )

    def open_dlg(e):
        page.dialog = dlg
        dlg.open = True
        page.update()

    page.add(
        ft.ElevatedButton("Open dialog", on_click=open_dlg),
    )

ft.app(target=main)
```

### Error message

```bash
N/A
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Created an issue: https://github.com/flet-dev/flet/issues/2520
Are there any other props of AlertDialog you are missing?

**Reply by vindevoy** [reply]
I tend to say: border.
